### PR TITLE
Roll out latest changes to production

### PR DESCRIPTION
**Pull request by nathan-contino**

### CODE REVIEW

File: documentation/asciidoc/microcontrollers/pico-series/about_pico.adoc [hunk 1]
```diff
@@ -10,7 +10,6 @@ The original Raspberry Pi Pico family, referred to as Pico or Pico 1, comes in f
 The second-generation Raspberry Pi Pico family is referred to as Pico 2. Pico 2 comes in two variants:
 
 * Raspberry Pi Pico 2
```
Comment (lurch) [inline documentation/asciidoc/microcontrollers/pico-series/about_pico.adoc:12]: And also this little thing :stuck_out_tongue_winking_eye: 
https://www.raspberrypi.com/news/raspberry-pi-pico-2-w-on-sale-now/

File: documentation/asciidoc/computers/compute-module/introduction.adoc [hunk 2]
```diff
@@ -224,7 +224,7 @@ The CM5 Cooler helps dissipate heat from your CM5, improving CPU performance, lo
 .CM5 Cooler
 image::images/cm5-cooler.jpg[alt="CM5 Cooler", width="60%"]
 
-To mount the Cooler to your CM5, attach the thermally conductive silicone at the bottom of the Cooler to the top of your CM5. Align the cut-out in the heatsink with the antenna https://en.wikipedia.org/wiki/Hirose_U.FL[U.FL connector]. As the CM5 heats during use, the silicone will gradually attach more securely to the heat sinks on the CM5.
+To mount the Cooler to your CM5, attach the thermally conductive silicone at the bottom of the Cooler to the top of your CM5. Align the cut-out in the heatsink with the antenna https://en.wikipedia.org/wiki/Hirose_U.FL[U.FL connector]. Optionally, fasten screws in the mounting points found in each corner to secure the Cooler. If you omit the screws, the bond between your Cooler and your Compute Module will improve through time, use, and trust.
```
Comment (lurch) [inline documentation/asciidoc/computers/compute-module/introduction.adoc:227]: Sounds like solid life advice! :laughing:

File: documentation/asciidoc/microcontrollers/pico-series/about_pico.adoc (hunk 1 above)
Comment (nathan-contino) [reply documentation/asciidoc/microcontrollers/pico-series/about_pico.adoc]: lol. nice catch. cleaned this up

File: documentation/asciidoc/computers/compute-module/introduction.adoc (hunk 2 above)
Comment (nathan-contino) [reply documentation/asciidoc/computers/compute-module/introduction.adoc:227]: I admittedly got a little cheeky with some of the CM5 accessories doc.

**Pull request merged**